zlp-scheduler/features/step_definitions/admin_steps.rb

```diff
@@ -25,4 +25,48 @@
 When /^I delete the new admin$/ do
     user = User.find_by(:email => "blah@tamu.edu")
     click_link("Delete", :href => "/admins/#{user.id}")
+end
+
+When /I click the label "(.*)"/ do |text|
+   find('label', :text => text).click 
+end
+
+When /I want to delete specific student/ do
+    
+    user = User.find_by(:cohort_id => @cohort.id)
+    
+    click_link("Delete", :href => "/users/#{user.id}")
+end
+
+When /I want to delete specific cohort/ do
+    click_link("Delete Current Cohort", :href => "/cohorts/#{@cohort.id}")
+end
+
+When /I edit "(.*)" in the page/ do |word| 
+   
+   dic = {"firstname"=>"user_firstname", "lastname"=>"user_lastname", "valid-email"=>"user_email","invalid-email"=>"user_email","existed-email"=>"user_email","Uin"=>"user_uin"}
+   val = {"firstname"=>"Corgi_Dog", "lastname"=>"Corgi_M", "valid-email"=>"corgi_mocha@tamu.edu","invalid-email"=>"corgi_mocha@gmail.com","existed-email"=>@user.email,"Uin"=>"999999999"}
+   fill_in dic[word], with: val[word]
+   @route = current_path
+   
+end
+
+When /I should remain in the same page/ do
+    expect(current_path).to eq @route
+end
+
+When /I click confirm in dialog/ do
```

Comment:
Did you end up using this step? If not, could we delete it since it would be unused?

Reply:
Hi, I didn't use it and I just deleted it! Thanks for advice!